### PR TITLE
the 3D doc tangziwen

### PR DESCRIPTION
**Pull request by tangziwen**

### CODE REVIEW

**Comment by pandamicro** [comment]
Please assign someone to review the doc, thanks

File: cocos/3d/CCAABB.h [hunk 1]
```diff
@@ -36,7 +36,10 @@ NS_CC_BEGIN
  */
 
 /**
- * Axis Aligned Bounding Box (AABB), usually caculate some rough but fast collision detection.
+ * @class AABB
+ * @brief  @~english Axis Aligned Bounding Box (AABB), usually caculate some rough but fast collision detection.
```
Comment (super626) [inline cocos/3d/CCAABB.h]: , is usually used to detect the collision at early step.

File: cocos/3d/CCAABB.h [hunk 2]
```diff
@@ -75,47 +91,85 @@ class CC_DLL AABB
...
+     * verts[4] : 右上后
+     * verts[5] : 右下后
+     * verts[6] : 左下后
+     * verts[7] : 左上后
      */
     void getCorners(Vec3 *dst) const;
 
     /**
-     * Tests whether this bounding box intersects the specified bounding object.
+     * @~english Tests whether this bounding box intersects the specified bounding object.
```
Comment (super626) [inline cocos/3d/CCAABB.h]: intersects with

File: cocos/3d/CCAnimate3D.h [hunk 3]
```diff
@@ -91,32 +111,88 @@ class CC_DLL Animate3D: public ActionInterval
...
+     * @~english Set the blend weight
+     * @~chinese 设置动画混合权重
+     * @param weight @~english The blend weight, the blend weight must be positive
+     * @~chinese 权重，权值必须为正
+     */
     void setWeight(float weight);
     
-    /**get & set origin interval*/
+    /**
+     * @~english Set origin interval
```
Comment (super626) [inline cocos/3d/CCAnimate3D.h]: It is internally used currently. For example, Sprite3D is playing an Animate3D, and then another Animate3D is going to be played. This function will be used to make the transition from on Animate3D to another smoothly.

File: cocos/3d/CCAnimate3D.h [hunk 4]
```diff
@@ -91,32 +111,88 @@ class CC_DLL Animate3D: public ActionInterval
...
+     * @~english Set the animate quality
+     * @~chinese 设置动画质量
+     * @param quality @~english The animate quality
+     * @~chinese 动画质量的枚举
+     * */
     void setQuality(Animate3DQuality quality);
     
-    /**get animate quality*/
+    /**
+     * @~english Get animate quality
```
Comment (super626) [inline cocos/3d/CCAnimate3D.h]: see also Animate3DQuality

Comment (super626) [inline cocos/3d/CCAnimate3D.h]: High quality means more cpu time

File: cocos/3d/CCMeshSkin.h [hunk 5]
```diff
@@ -52,28 +54,88 @@ class CC_DLL MeshSkin: public Ref
     friend class Mesh;
 public:
     
-    /**create a new meshskin if do not want to share meshskin*/
+    /**
+     * @~english Create a new meshskin if do not want to share meshskin
+     * @param skeleton @~english The specified skeleton.
+     */
     static MeshSkin* create(Skeleton3D* skeleton, const std::string& filename, const std::string& name);
```
Comment (super626) [inline cocos/3d/CCMeshSkin.h:61]: @param

File: cocos/3d/CCPlane.h [hunk 6]
```diff
@@ -29,16 +29,35 @@
...
-    IN_PLANE,
+    /** @~english The Point which in the plane. @~chinese 点在平面上*/
+    IN_PLANE, 
+    /**
+     * @~english The point which in the front side of the plane, which means the dot product for point and normal is greater  than 0 
+     * @~chinese 点在平面的正面，即点向量（原点到该点的向量）与平面法线的点积大于零
+     */
     FRONT_PLANE,
+    /**
+     * @~english The point which in the front side of the plane, which means the dot product for point and normal is less  than 0
```
Comment (super626) [inline cocos/3d/CCPlane.h]: in the back side of plane

File: cocos/3d/CCRay.h [hunk 7]
```diff
@@ -38,70 +38,113 @@ NS_CC_BEGIN
  */
 
 /**
- * @brief Ray is a line with one end. usually use it to check intersects with some object,such as Plane, OBB, AABB
+ * @class Ray
+ * @brief @~english Ray is a line with one end. usually use it to check intersects with some object,such as Plane, OBB, AABB
```
Comment (super626) [inline cocos/3d/CCRay.h]: Ray is line with a start position.

File: cocos/3d/CCMeshSkin.h (hunk 5 above)
Comment (tangziwen) [inline cocos/3d/CCMeshSkin.h:61]: this method in fact, has not been implemented yet.

**Comment by super626** [comment]
@pandamicro 
@andyque 
Done.

**Pull request merged**